lib/tom_queue/delayed_job/ack_work_plugin.rb

```diff
@@ -0,0 +1,15 @@
+require 'active_support/concern'
+
+module TomQueue
+  module DelayedJob
+    class AckWorkPlugin < Delayed::Plugin
+      callbacks do |lifecycle|
+        lifecycle.after(:perform) do |_, job|
```

Comment:
We now hook into the callbacks around the run method which are called here - https://github.com/collectiveidea/delayed_job/blob/71f1d5faf934d3057abca942f0d410327bc69087/lib/delayed/worker.rb#L301